packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -171,21 +122,22 @@ const MediaReplaceFlow = ( {
 								} }
 							/>
 						</MediaUploadCheck>
-						{ onSelectURL && (
-							<MenuItem
-								icon={ link }
-								onClick={ () =>
-									setShowURLInput( ! showURLInput )
-								}
-								aria-expanded={ showURLInput }
-							>
-								<div> { __( 'Insert from URL' ) } </div>
-							</MenuItem>
-						) }
 					</NavigableMenu>
-					{ showURLInput && (
+					{ onSelectURL && (
 						<div className="block-editor-media-flow__url-input">
-							{ urlInputUIContent }
+							<span className="block-editor-media-replace-flow__image-url-label">
+								{ __( ' Current media URL:' ) }
```

Comment:
Not sure the leading space is intentional?

```suggestion
								{ __( 'Current media URL:' ) }
```

If you did need a bit more space, I would think style-applied padding would be more reliable.

Reply:
Oh surely I didn't mean to use a space instead of padding :)))) good catch!